.github/copilot-instructions.md

```
@@ -58,0 +1,194 @@
+
+# Copilot & AI Agent Instructions for Hatchgrid
+
+## 🏗️ Big Picture Architecture
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)